serverlessworkflow/modules/ROOT/pages/getting-started/modules/create-your-first-workflow-service.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+:description: Creating first workflow service
+:keywords: creating, serverless, workflow
+:navtitle: Creating your first workflow service
+:page-aliases:
+
+[id="create-your-first-workflow-service_{context}"]
+= Creating your first workflow service
+
+As a developer, you can use {context} and create a `Hello World` application, which includes the following procedures:
+
+* xref:getting-started/modules/proc-boostrapping-the-project.adoc[Bootstrapping a project]
+* xref:getting-started/modules/proc-creating-workflow.adoc[Creating a workflow]
+* xref:getting-started/modules/proc-running-application.adoc[Running your workflow application]
+* xref:getting-started/modules/proc-testing-application.adoc[Testing your workflow application]
```

Comment:
I think here we just need links within the same document to quickly "scroll down", not links to separate sections. Linked documents are inserted in this main document anyway, so we just need a quick link to the relevant section within the same document. Now it sends the reader to the single document, e.g. `proc-boostrapping-the-project.adoc` and it doesn't stay within the `create-your-first-workflow-service.adoc`.